### ISSUE TITLE
Any Mouse Event is not working on Wpf-VlcControl

### ISSUE THREAD

**Issue by dhpanteon** [ZeBobo5/Vlc.DotNet#491]
I have an issue  about Vlc.DotNet.

# Generic information
- **Issue : Any Mouse Event is not working on Wpf-VlcControl**
- **Vlc.DotNet version** : 2.2.1
- **Vlc.DotNet project used** : Xamarin-WPF
- **libvlc version** : x86
- **.net version** : 4.7.1
- **Project language** : C#
- **Project build architecture** : x86
- **Operating system** : Windows x64

# Summary

I have Added VlcControl from codebehind. It's not firing any mouse event (i.e. mousemove, mousedown, click etc..) . But i found that keyboard events are firing perfectly.

Basically i want to open window's Contextmenu on right click of it. (Setting Vlccontrol's ContextMenu is also not working.)

Setting VlcControl.Background or VlcControl.MediaPlayer.BackColor is also not working.

(Note: i have added VlcControl on System.Windows.Controls.Grid using ViewRenderer on wpf.)

Currently i am using [Mouse Hook](https://www.codeproject.com/Articles/7294/Processing-Global-Mouse-and-Keyboard-Hooks-in-C) as a patch solution.

**Comment by dilandau2001** [comment]
Have you check what was reviewed here #473.
There was another user having similar problem

**Comment by dhpanteon** [comment]
The Property called IsMouseInputEnabled and IsKeyInputEnabled is not available on vlc lib.

**Comment by mfkl** [comment]
> Vlc.DotNet version : 2.2.1

Update

**Comment by dhpanteon** [comment]
I am already using latest version 
Vlc.DotNet.Wpf 2.2.1: https://www.nuget.org/packages/Vlc.DotNet.Wpf

**Comment by mfkl** [comment]
Install the pre-release version for all Vlc.DotNet packages

**Comment by dhpanteon** [comment]
There is a lots of changes in pre-release version. can anyone tell the me the basic thing setting VlcLibDirectory is missing. 

How do i set it?

**Comment by mfkl** [comment]
Check the samples

**Comment by jeremyVignelles** [comment]
Use the WinForms version instead of the Wpf version, there are less breaking changes that way. 3.0 is the version I recommend because a lot of improvements have been made

**Comment by dhpanteon** [comment]
I have upgrade to 3.0.0-develop322 version. Now video playing is totally stooped. I have tried both WPF and winform but still not working.  Also upgrading to 3.0.0 version produce new error for System.Runtime version 4.1.2.0. I have manually change the version number in App.config file to solve this problem.

Even Samples are to old on WPF version. There is no release-note to track this pre-release version are enough stable or not. If there are lots of changes then you should make a stable release version.

Currently i am switching back to older version.  I can leave with [Mouse Hook](https://www.codeproject.com/Articles/7294/Processing-Global-Mouse-and-Keyboard-Hooks-in-C) Patch. Instead of spending to much time on this.

**Comment by jeremyVignelles** [comment]
I didn't take time to update the release note each time, I apologize for that.
Your System.Runtime issue is something I never encountered and that feels weird... you shouldn't have to do that.

However, I disagree with you : all the samples are up to date with the latest 3.0 develop branch. You can clone this repo and see by yourself.

If you encounter any error with the -develop version, chances are that you will have them when 3.0 is released.

By the way, you didn't mention which libvlc version you are using. Are you using 3.0.4? did you install it from NuGet?

**Comment by jeremyVignelles** [comment]
Hi, did you manage to fix this issue?

Feel free to contact me on gitter if you need help upgrading to 3.0.0

**Comment by dhpanteon** [comment]
Sorry for the delay. It solves my problem. (y)
